integration-cli/docker_cli_run_unix_test.go

```diff
@@ -322,9 +322,18 @@ func (s *DockerSuite) TestRunWithSwappiness(c *check.C) {
 func (s *DockerSuite) TestRunWithSwappinessInvalid(c *check.C) {
 	testRequires(c, memorySwappinessSupport)
 	out, _, err := dockerCmdWithError("run", "--memory-swappiness", "101", "busybox", "true")
-	if err == nil {
-		c.Fatalf("failed. test was able to set invalid value, output: %q", out)
+	c.Assert(err, check.NotNil)
+	expected := "Valid memory swappiness range is 0-100"
+	if !strings.Contains(out, expected) {
+		c.Fatalf("Expected output to contain %q, not %q", out, expected)
 	}
+
+	out, _, err = dockerCmdWithError("run", "--memory-swappiness", "101", "busybox", "true")
```

Comment:
It is my mistake. It should be "dockerCmdWithError("run", "--memory-swappiness", "-1", "busybox", "true")".